pmd-java/src/main/resources/category/java/codestyle.xml

```diff
@@ -48,6 +48,26 @@ public abstract class Foo { // should be AbstractFoo
         </example>
     </rule>
 
+    <rule name="ArrayInitializationVerboseness"
```

Comment:
I would suggest naming the rule "UseArrayInitializer" or similar. "ArrayInitializationVerboseness" seems a bit vague to me